lib/src/components/NxTabs/__tests__/NxTabs.test.tsx

```diff
@@ -5,103 +5,76 @@
-      <NxTabs onTabSelect={() => {}} activeTab={-1} className="test">
-        <NxTabList>
-          <NxTab>Tab 0</NxTab>
-        </NxTabList>
-        <NxTabPanel>Content 0</NxTabPanel>
-      </NxTabs>
-    );
+  it('merges any passed in className', function() {
+    const tabsWithAddedClassName = renderEl({ activeTab: -1, className: 'foo' });
+    const tabs = renderEl({ activeTab: -1 })!;
+
+    expect(tabsWithAddedClassName).toHaveClass('foo');
 
-    expect(component).toHaveClassName('nx-tabs test');
+    for (const cls of Array.from(tabs.classList)) {
+      expect(tabsWithAddedClassName).toHaveClass(cls);
+    }
   });
 
-  it('renders a generated id', function () {
-    const component = shallow(
-      <NxTabs onTabSelect={() => {}} activeTab={-1}>
-        <NxTabList>
-          <NxTab>Tab 0</NxTab>
-        </NxTabList>
-        <NxTabPanel>Content 0</NxTabPanel>
-      </NxTabs>
-    );
+  it('renders with a unique id each time', function() {
+    const firstTabs = renderEl({ activeTab: 0 })!;
+    const secondTabs = renderEl({ activeTab: 0 })!;
 
-    expect(component.prop('id')).toMatch(/^nx-tabs-\d+$/);
+    expect(firstTabs.id).not.toEqual(secondTabs.id);
   });
 
-  it('renders using as specific id', function () {
-    const component = shallow(
-      <NxTabs onTabSelect={() => {}} activeTab={-1} id="my-tabs">
-        <NxTabList>
-          <NxTab>Tab 0</NxTab>
-        </NxTabList>
-        <NxTabPanel>Content 0</NxTabPanel>
-      </NxTabs>
-    );
+  it('renders a generated id', function() {
+    const component = renderEl({ activeTab: -1 })!;
+    expect(component.id).toMatch(/^nx-tabs-\d+$/);
+  });
 
-    expect(component.prop('id')).toMatch('my-tabs');
+  it('renders using as specific id', function() {
+    const commponent = renderEl({
+      id: 'my-tabs',
+      activeTab: -1
+    })!;
+    expect(commponent.id).toMatch('my-tabs');
   });
 
-  it('renders no tab contents when none are active', function () {
-    const component = mount(
-      <NxTabs onTabSelect={() => {}} activeTab={-1}>
-        <NxTabList>
-          <NxTab>Tab 0</NxTab>
-        </NxTabList>
-        <NxTabPanel>Content 0</NxTabPanel>
-      </NxTabs>
-    );
+  it('renders no tab contents when none are active', function() {
+    const { getByRole, queryByRole } = quickRender({ activeTab: -1 });
 
-    expect(component.find(NxTab)).not.toHaveClassName('active');
-    expect(component.find(NxTabPanel)).toBeEmptyRender();
+    expect(getByRole('tab').classList.contains('active')).toBeFalsy();
+    expect(queryByRole('tabpanel')).not.toBeInTheDocument();
   });
 
-  it('renders no tab contents when no active tab is specified', function () {
-    const component = mount(
-      <NxTabs onTabSelect={() => {}}>
-        <NxTabList>
-          <NxTab>Tab 0</NxTab>
-        </NxTabList>
-        <NxTabPanel>Content 0</NxTabPanel>
-      </NxTabs>
-    );
+  it('renders no tab contents when no active tab is specified', function() {
```

Comment:
This doesn't really seem like a different test from the one above.  `activeTab=-1` and no `activeTab` at all seem like two different ways to specify the same thing.  Maybe these tests should be merged.

Reply:
> This doesn't really seem like a different test from the one above. `activeTab=-1` and no `activeTab` at all seem like two different ways to specify the same thing. Maybe these tests should be merged.

Fixed: fc60025ae197c6b928bcd7b180351ad7dd1dc690